src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -276,6 +278,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _core = _interactivity.Core();
 
+        // If high contrast mode was changed, update the appearance appropriately.
+        _accessibilitySettings.HighContrastChanged([this](const Windows::UI::ViewManagement::AccessibilitySettings& a11ySettings, auto&&) {
+            _core.SetHighContrastInfo(a11ySettings.HighContrast());

```

Comment:
I feel like this only gets called when it _changes_. Where does it get the initial value, on launch?

Reply:
Good catch!